mkdocs.yml

```
@@ -1,7 +1,7 @@
 site_name: LibreTuya
 docs_dir: .
 
-site_url: https://kuba2k2.github.io/libretuya/
+site_url: https://docs.libretuya.ml//
```

Oops, a double trailing slash.

```suggestion
site_url: https://docs.libretuya.ml/
```